databricks/koalas/base.py

```diff
@@ -767,6 +767,9 @@ def is_monotonic(self) -> bool:
             which is potentially expensive. In case of multi-index, all data are
             transferred to single node which can easily cause out-of-memory error currently.
 
+        .. note:: Disable the `spark.sql.optimizer.nestedSchemaPruning.enabled`
+            for multi-index if you're using Koalas < 1.7.0 with PySpark 3.1.1.
```

Comment:
Ideally we should mention the JIRA issue link (`SPARK-XXXXX`). Shall we make a reproducer, report to Apache Spark and add the issue link here?

Reply:
Sure, let me create & add a link. Thanks!